collector/init.go

```diff
@@ -2,406 +2,418 @@ package collector
 
 import "github.com/alecthomas/kingpin/v2"
 
-// collectorInit represents the required initialisation config for a collector.
-type collectorInit struct {
+// Initializer represents the required initialisation config for a collector.
```

Comment:
This file had the most functional change, also not very happy with the name, can change to CollectorInit though linting complains about that.